app/src/test/java/ch/hikemate/app/authentication/FirebaseAuthRepositoryTest.kt

```diff
@@ -372,9 +375,8 @@ class FirebaseAuthRepositoryTest {
       every { mockDocument.delete() } returns mockVoidTask
 
       val onSuccess: () -> Unit = mockk(relaxed = true)
-      val mockActivity: Activity = mockk(relaxed = true)
 
-      repository.deleteAccount("password", mockActivity, onSuccess, mockOnError)
+      repository.deleteAccount("password", context, this, onSuccess, mockOnError)
 
       verify { mockOnError(any()) }
```

Comment:
Question: Does it make sense to test here whether to test if the re-authentication was called? Or is too much mocked?

Reply:
Seems good. I added in both tests.